json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -4,6 +4,10 @@
 class Jetpack_JSON_API_Sync_Endpoint extends Jetpack_JSON_API_Endpoint {
```

Comment:
It feels like this should be an abstract base class, rather than the other classes extending an endpoint that has its own, totally separate functionality.